integration/testcmd/mountkexec/uinit/mountkexec.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+	"os/exec"
+
+	"github.com/u-root/webboot/pkg/mountkexec"
+	"golang.org/x/sys/unix"
+)
+
+func main() {
+	var tests = []struct {
+		iso string
+		dir string
+	}{
+		{"testIso", ""},
+		{"non-existentfile", "/tmp/mountDir"},
+		{"testIso", "/tmp/mountDir"},
+	}
+
+	for _, test := range tests {
+		if err := mountkexec.MountISO(test.iso, test.dir); err != nil {
+			log.Print(err)
+		}
+	}
+	o, err := exec.Command("ls", "/tmp/mountDir").CombinedOutput()
```

Comment:
This assumes that the last mount works. Put this inside the for loop instead, and I suggest you name the subtests as you run them, and in the expect, check for the name as part of the string to be expected. You want to know that the "non-existentfile" test really was the test that produced "error setting loop device:open non-existentfile: no such file or directory", and not another test. Don't just rely on the order of the tests.

Reply:
Done